common/static/sass/edx-pattern-library-shims/base/_variables.scss

```diff
@@ -178,6 +178,22 @@ $btn-brand-active-background:             $uxpl-blue-base !default;
 $btn-brand-disabled-background:           #f2f3f3 !default;
 $btn-brand-disabled-color:                #676666 !default;
 
+// Upgrade button
+$btn-upgrade-border-color:                $uxpl-green-base !default;
```

Comment:
I think @andy-armstrong agrees that that these variables do not belong in this file, but he does want to keep generic names.